tests/features/project.feature

```diff
@@ -7,6 +7,8 @@ Feature: Project content creation
   @javascript
   Scenario: Creation of a Project content through the UI.
     Given I am logged in as a user with the "create oe_project content, access content, edit own oe_project content, view published skos concept entities, manage corporate content entities" permission
+    And a dummy "My stakeholder 1" stakeholder
```

Comment:
with TableNode you will call this in a table like below for documents instead of duplicate lines

Reply:
Previously fixed